Dockerfile

```diff
@@ -16,11 +16,15 @@ RUN apt-get update && apt-get install -y \
   libpq-dev \
   build-essential \
   whois \
+  zlib1g-dev \
   && rm -rf /var/lib/apt/lists/*
 
-RUN pecl install yaml-1.3.1
+RUN yes '' | pecl install yaml-1.3.1
```

Comment:
Just asking, what that `yes ' ' | ` does?

Reply:
"pecl install yaml" started to ask about a custom libname or "yes" to default.
pecl doesn't have an "-y" option, so I used the "yes" command, this "yes ' ' | " is a look foward combo command ( I beleave that it have another name, but I don't know. ) ,  it send an "yes" to every terminal ask made by the next command.

Reply:
Man pages says:

       yes - output a string repeatedly until killed

SYNOPSIS
       yes [STRING]...
       yes OPTION

DESCRIPTION
       Repeatedly output a line with all specified STRING(s), or 'y'.

       --help display this help and exit

       --version
              output version information and exit